src/github_profile.py

```diff
@@ -4,7 +4,7 @@
 def generate_profile(theme, **kwargs):
     """Generate profile readme from theme and kwargs"""
     # Read theme profile
-    with open(f"src/themes/{theme}/profile.txt") as f:
+    with open(f"../src/themes/{theme}/profile.txt") as f:
```

Comment:
⚪ Isn't it better to run the App on top of src folder and delete the `../` part of URL?